worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/RegionProtectionListener.java

```diff
@@ -261,8 +261,8 @@ public void onUseBlock(final UseBlockEvent event) {
                 canUse = query.testBuild(BukkitAdapter.adapt(target), associable, combine(event, Flags.CHEST_ACCESS));
                 what = "open that";
 
-            /* Beds */
-            } else if (Materials.isBed(type)) {
+            /* Beds or Respawn Anchor */
+            } else if (Materials.isBed(type) || type == Material.RESPAWN_ANCHOR) {
```

Comment:
That's not the perfect solution for that.
Respawn Anchors has to be loaded by players with 4 glowstones. Only with full loaded anchors it's possible to set the spawn point.

And the message "sleep" doesn't match it. However I would let that inside the sleep flag because it's close enough to "set your spawnpoint"